### PR TITLE
IVORY-326: CITES screens

### PR DESCRIPTION
**Pull request by duncannixon**
This change adds 3 new screens that determine whether the user needs to apply for a CITES permit when they sell or hire out their item.

### CODE REVIEW

File: server/routes/where-is-item.route.js [hunk 1]
```diff
@@ -1,42 +1,64 @@
...
+const RedisService = require('../services/redis.service')
+const { Options, Paths, RedisKeys, Views } = require('../utils/constants')
+const { buildErrorSummary, Validators } = require('../utils/validation')
 
 const handlers = {
-  get: (request, h) => {
+  get: async (request, h) => {
     return h.view(Views.WHERE_IS_ITEM, {
-      ..._getContext()
+      ...(await _getContext(request))
```
Comment (Jozzey) [inline server/routes/where-is-item.route.js:10]: Don't think you need the await or to pass the request for _getContext.

File: server/routes/where-is-item.route.js [hunk 2]
```diff
@@ -1,42 +1,64 @@
...
-  post: (request, h) => {
+  post: async (request, h) => {
     const payload = request.payload
     const errors = _validateForm(payload)
 
     if (errors.length) {
       return h
         .view(Views.WHERE_IS_ITEM, {
-          ..._getContext(),
+          ...(await _getContext(request)),
```
Comment (Jozzey) [inline server/routes/where-is-item.route.js:21]: Don't think you need the await or to pass the request for _getContext.

**Review by Jozzey — CHANGES_REQUESTED** [review]
Nice piece of work. Just noticed a few things that are probably left overs from code you've reused

File: server/routes/intention-for-item.route.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  },
+
+  post: async (request, h) => {
+    const payload = request.payload
+    const errors = _validateForm(payload)
+
+    if (errors.length) {
+      return h
+        .view(Views.INTENTION_FOR_ITEM, {
+          ...(await _getContext(request)),
```
[the remark this answers is not in the document]
Comment (duncannixon) [reply server/routes/intention-for-item.route.js:21]: yeah you're right, well spotted

File: server/routes/where-is-item.route.js (hunk 2 above)
Comment (duncannixon) [reply server/routes/where-is-item.route.js:21]: Fixed.

File: server/routes/intention-for-item.route.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const RedisService = require('../services/redis.service')
+const { Paths, RedisKeys, SaleIntention, Views } = require('../utils/constants')
+const { buildErrorSummary, Validators } = require('../utils/validation')
+
+const handlers = {
+  get: async (request, h) => {
+    return h.view(Views.INTENTION_FOR_ITEM, {
+      ...(await _getContext(request))
```
[the remark this answers is not in the document]
Comment (duncannixon) [reply server/routes/intention-for-item.route.js:10]: Fixed.

**Review by Jozzey — APPROVED** [review]

**Pull request merged**